# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: LC_NUMERIC: copy de_CH into it_CH
# DP: Dpatch author: Denis Barbier <barbier@debian.org>
# DP: Patch author: Denis Barbier
# DP: Upstream status: not submitted
# DP: Date: 2006-10-02

Index: b/localedata/locales/it_CH
===================================================================
--- a/localedata/locales/it_CH	2008-08-08 19:08:32.000000000 +0200
+++ b/localedata/locales/it_CH	2008-08-08 19:08:38.000000000 +0200
@@ -57,9 +57,7 @@
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point             "<U002C>"
-thousands_sep             "<U0027>"
-grouping                  3;3
+copy  "de_CH"
 END LC_NUMERIC
 
 LC_TIME
